code/modules/clothing/under/jobs/medsci.dm

```diff
@@ -171,10 +181,17 @@
 	desc = "It's made of a special fiber that provides minor protection against biohazards. It has a dark blue cross on the chest denoting that the wearer is a trained paramedic."
 	icon_state = "paramedic_skirt"
 	item_state = "w_suit"
+	item_color = "paramedic_skirt"
 	body_parts_covered = CHEST|GROIN|ARMS
 	can_adjust = FALSE
 	fitted = FEMALE_UNIFORM_TOP
 
+/obj/item/clothing/under/rank/medical/paramedic/skirt/dark
```

Comment:
if you want paramedics to spawn with the dark variant of the skirt, this will HAVE to be named `/obj/item/clothing/under/rank/medical/paramedic/skirt` or else it will give them the one that is named that. If you want that to be, name this that and name the other one `/obj/item/clothing/under/rank/medical/paramedic/skirt/light` which will also be in line the regular jumpsuit naming scheme.

Reply:
fixed